docs/usage_guide/advanced_usage/index.rst

```diff
@@ -30,42 +31,54 @@ Modin also supports these experimental APIs on top of pandas that are under acti
 - :py:func:`~modin.experimental.pandas.read_pickle_distributed`  -- read multiple files in a directory
 - :py:meth:`~modin.experimental.pandas.DataFrame.to_pickle_distributed` -- write to multiple files in a directory
 
+DataFrame partitioning API
+--------------------------
+
+Modin DataFrame provides an API to directly access partitions: you can extract physical partitions from
+a :py:class:`~modin.pandas.dataframe.DataFrame`, modify their structure by reshuffling or applying some
+functions, and create a DataFrame from those modified partitions. Visit
+:doc:`pandas partitioning API </flow/modin/distributed/dataframe/pandas>` documentation to learn more.
+
 Modin Spreadsheet API: Render Dataframes as Spreadsheets
 --------------------------------------------------------
-The Spreadsheet API for Modin allows you to render the dataframe as a spreadsheet to easily explore 
-your data and perform operations on a graphical user interface. The API also includes features for recording 
-the changes made to the dataframe and exporting them as reproducible code. Built on top of Modin and SlickGrid, 
-the spreadsheet interface is able to provide interactive response times even at a scale of billions of rows. 
+
+The Spreadsheet API for Modin allows you to render the dataframe as a spreadsheet to easily explore
+your data and perform operations on a graphical user interface. The API also includes features for recording
+the changes made to the dataframe and exporting them as reproducible code. Built on top of Modin and SlickGrid,
+the spreadsheet interface is able to provide interactive response times even at a scale of billions of rows.
 See our `Modin Spreadsheet API documentation`_ for more details.
 
-.. figure:: ../img/modin_spreadsheet_mini_demo.gif
+.. figure:: /img/modin_spreadsheet_mini_demo.gif
    :align: center
    :width: 650px
-   :height: 350px   
+   :height: 350px
 
 Progress Bar
 ------------
-Visual progress bar for Dataframe operations such as groupby and fillna, as well as for file reading operations such as 
+
+Visual progress bar for Dataframe operations such as groupby and fillna, as well as for file reading operations such as
 read_csv. Built using the tqdm library and Ray execution engine. See `Progress Bar documentation`_ for more details.
```

Comment:
link to tqdm: https://github.com/tqdm/tqdm

Reply:
added